rtlib/string.hh

```diff
@@ -56,9 +82,8 @@ class String {
         Block &operator=(const Block &b);
 
      public:
-        uint32_t ref_count;
-      // enum { size = 27 };
-      enum { size = 59 };
+      uint32_t ref_count;
+      enum { size = 59 };  // total size of object should be 64 bytes
```

Comment:
I don't understand you comment. Are you saying that 59 should be changed into 64? Do you know what the missing 5 bytes are used for? It was already increased from originally 27, wasn't it?

Reply:
No, I meant that the total size of a `Block` object should be 64 bytes (I guess for maximum efficiency in the memory pool?):

`ref_count` (4 bytes) + `pos` ( 1 byte) + `array` (59 bytes) = 64 bytes total.

I adjusted the comment to make this more clear.